src/main/scala/org/broadinstitute/dsde/firecloud/service/TrialService.scala

```diff
@@ -20,12 +20,13 @@ import spray.json.DefaultJsonProtocol._
 
 import scala.concurrent.{ExecutionContext, Future}
 
+// TODO: Contain userEmail in value class for stronger type safety without incurring performance penalty
 object TrialService {
   sealed trait TrialServiceMessage
-  case class EnableUsers(userInfo:UserInfo, users: Seq[String]) extends TrialServiceMessage
-  case class DisableUsers(userInfo:UserInfo, users: Seq[String]) extends TrialServiceMessage
-  case class EnrollUser(userInfo:UserInfo) extends TrialServiceMessage
-  case class TerminateUsers(userInfo:UserInfo, users: Seq[String]) extends TrialServiceMessage
+  case class EnableUsers(managerInfo: UserInfo, userEmails: Seq[String]) extends TrialServiceMessage
+  case class DisableUsers(managerInfo: UserInfo, userEmails: Seq[String]) extends TrialServiceMessage
+  case class EnrollUser(managerInfo: UserInfo) extends TrialServiceMessage
```

Comment:
`enrollUser` is different: end users enroll themselves, so this should be `userInfo`. For enroll, disable, and terminate, it's a manager making the call, so `managerInfo` is correct there.